Tasks/Maven/maventask.ts

```diff
@@ -272,6 +273,29 @@ function processMavenOutput(data) {
     }
 }
 
+// Identifies all maven modules below the root by the presence of a pom.xml file and a /target/ directory
```

Comment:
As discussed offline, this needs a more explanatory name e.g. _findCandidateModules_.

The comment should also call out that we have no way of knowing which POMs are actually being built; we are assuming that the user/POM hasn't changed the default _target_ build directory (which is possible), and identifying possible modules location by the presence of a pom.xml file and a /target/ directory.